packages/design-system-tokens/src/tokens/Theme.medicare.json

```diff
@@ -1158,6 +1158,16 @@
               "codeSyntax": {}
             }
           }
+        },
+        "semibold": {
```

Comment:
this is what i'm talking about above - the "bold" token is mapped to the 600 weight, meaning there is no true bold/700 weight font in mgov